app/views/errors/not_found.html.erb

```diff
@@ -10,7 +10,7 @@
       If you pasted the web address, check you copied the entire address.
     </p>
     <p class="govuk-body">
-      If the web address is correct or you selected a link or button and you need to speak to someone about this problem, contact the Becoming a Teacher team: <%= bat_contact_mail_to subject: "Page not found" %>
```

Comment:
@Kateharries @simonwhatley A line of text here is different in the [design history](https://bat-design-history.netlify.app/find-teacher-training/updating-error-pages/) and the [prototype](https://publish-courses-prototype.herokuapp.com/404), which should it be?

> If you typed **a** web address, check it is correct.
or 
> If you typed **the** web address, check it is correct.

Reply:
Never mind, I think it's `a` in both of them. I'm confusing myself with all my tabs 😄